dashboard/serializers.py

```diff
@@ -0,0 +1,52 @@
+"""
+Provides functionality for serializing a ProgramEnrollment for the ES index
+"""
+from decimal import Decimal
+
+
+class UserProgramSerializer:
+    """
+    Provides functions for serializing a ProgramEnrollment for the ES index
+    """
+    @staticmethod
+    def calculate_certificate_grade_average(certificates):
+        """
+        Calculates an average grade (integer) from a list of certificates with <1 decimal grades
+        """
+        return None if len(certificates) == 0 else round(
+            (sum([Decimal(certificate['grade']) for certificate in certificates]) / len(certificates)) * 100
```

Comment:
Can you change the square brackets in `[Decimal(..) for cert in certificates]` to parentheses, since `sum` will evaluate on all iterables?